src/PRo3D.Viewer/RemoteApi.fs

```diff
@@ -759,6 +811,57 @@ module RemoteApi =
                         path "/queryAnnotationAsObj" >=> request (queryAnnotationAsObj api)
                     ]
                 )
+                prefix "/exp" >=> 
+                    choose [
+                        path "/greet" >=> 
```

Comment:
is this just testing code or a real endpoint

Reply:
testing code